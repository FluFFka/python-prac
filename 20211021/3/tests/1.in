2
а б в г д е ёё жж ii oo qwe,qw'lewyo<F5>i64jtmphgy7h3w69gi6n8hp3q6b7y9a4po86by9<F5>8w5
1155 44 6- +    q

