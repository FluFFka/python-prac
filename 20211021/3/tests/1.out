by gi hp ii oo po qw жж ёё 
